tap_facebook/__init__.py

```diff
@@ -210,16 +250,20 @@ def do_request():
 
 def get_start(state, tap_stream_id, bookmark_key):
     current_bookmark = singer.get_bookmark(state, tap_stream_id, bookmark_key)
-    LOGGER.info("found current bookmark %s", current_bookmark)
     if current_bookmark is None:
-        LOGGER.info("using start_date instead...%s", CONFIG['start_date'])
-        return CONFIG['start_date']
-    return current_bookmark
+        if tap_stream_id in INCREMENTAL_STREAMS:
```

Comment:
It seems odd to have this switch here, when we have a subclass to represent incremental streams anyway. Is there a way to refactor it so the Stream class is used here? If it's really complicated, it's not a big deal.

Reply:
Good point, yeah it could be checking the type here instead. I'll give that a shot.